### PR TITLE
chore: upgrade to celestia-core@v1.3.3-tm-v0.34.20

### PR DESCRIPTION
**Pull request by rootulp**
Upgrade to https://github.com/celestiaorg/celestia-core/releases/tag/v1.3.3-tm-v0.34.20

### CODE REVIEW

**Review by evan-forbes — APPROVED** [review]

**Comment by evan-forbes** [comment]
pre-approved pending the CI not melting

**Comment by rootulp** [comment]
@evan-forbes or @liamsi may I please have write access on this repo?

Screen Shot 2022-08-15 at 11 56 05 AM

**Comment by evan-forbes** [comment]
I can't grant access here.

fwiw, while there's no reason not to upgrade (so we should proceed), there's also no reason (other than semantics) to upgrade since we're replacing the same version of tendermint in the app and this should replace this version as well iirc. Also we made no changes relevant to the sdk :shrug:

**Pull request merged**
